intelligence/views.py

```
@@ -210,4 +217,274 @@
+        description="List all pattern analyses with optional filtering by patient and pattern type"
+    ),
+    retrieve=extend_schema(
+        summary="Get pattern analysis",
+        description="Retrieve a specific pattern analysis by ID"
+    )
+)
+class PatternAnalysisViewSet(viewsets.ReadOnlyModelViewSet):
+    """ViewSet for pattern analysis results"""
+    queryset = PatternAnalysis.objects.all().select_related('patient')
+    serializer_class = PatternAnalysisSerializer
+    permission_classes = [IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)